src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -150,92 +150,98 @@ class JesBackend extends Backend with LazyLogging {
     )
   }
 
-  override def setupCallEnvironment(call: Call, workflowDescriptor: WorkflowDescriptor): JesTaskExecutionContext = {
-    val callGcsPath = s"${workflowDescriptor.callDir(call)}"
-    JesTaskExecutionContext(GoogleCloudStoragePath(callGcsPath), JesConnection)
+  override def bindCall(workflowDescriptor: WorkflowDescriptor,
+                        call: Call,
+                        locallyQualifiedInputs: CallInputs,
+                        abortRegistrationFunction: AbortFunctionRegistration): BackendCall = {
+    JesBackendCall(this, workflowDescriptor, call, locallyQualifiedInputs, abortRegistrationFunction)
   }
 
-  override def executeCommand(instantiatedCommandLine: String,
-                              workflowDescriptor: WorkflowDescriptor,
-                              call: Call,
-                              backendInputs: CallInputs,
-                              scopedLookupFunction: ScopedLookupFunction,
-                              abortFunctionRegistration: AbortFunctionRegistration): Try[Map[String, WdlValue]] = {
+  override def execute(backendCall: BackendCall): Try[CallOutputs] = backendCall match {
```

Comment:
I think the confusion comes because there is an `execute` method on BackendCall as well as Backend.  I noticed that too and it crossed my mind that that might be refactored but I just haven't given thought to it beyond that